File: app/src/debug/java/com/nawin/androidmvparchitecture/BuildTypeConfig.java
package com.nawin.androidmvparchitecture;

import android.content.Context;

import com.facebook.stetho.Stetho;

import javax.inject.Inject;

/**
 * Created by nawin on 6/15/17.
 */

public class BuildTypeConfig {
    private final Context context;

    @Inject
    BuildTypeConfig(Context context) {
        this.context = context;
    }

    void set() {
        Stetho.initializeWithDefaults(context);
    }

}

File: app/src/debug/java/com/nawin/androidmvparchitecture/BuildTypeConfig.kt
package com.nawin.androidmvparchitecture

import android.content.Context
import com.facebook.stetho.Stetho
import javax.inject.Inject

/**
 * Created on 11/29/17.
 */
class BuildTypeConfig @Inject constructor(private val context: Context) {

     fun set() {
        Stetho.initializeWithDefaults(context)
    }
}